django/contrib/admin/static/admin/css/base.css

```diff
@@ -733,27 +768,27 @@ div.breadcrumbs a:focus, div.breadcrumbs a:hover {
 
 .viewlink, .inlineviewlink {
     padding-left: 16px;
-    background: url(../img/icon-viewlink.svg) 0 1px no-repeat;
+    mask-image: url(../img/icon-viewlink.svg) 0 1px no-repeat;
```

Comment:
Here and below – will also require further work.

Reply:
because shorthand properties...

Reply:
would `mask: url(../img/icon-viewlink.svg) 0 1px no-repeat; ` work?

Reply:
Same as above, I’ve never used `mask` but if it works that sounds great!